test/backup_fetch_handler_test.go

```diff
@@ -3,39 +3,40 @@ package test
 import (
 	"github.com/stretchr/testify/assert"
 	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/testtools"
 	"testing"
 )
 
 func TestGetRestoredBackupFilesToUnwrap_SimpleFile(t *testing.T) {
 	sentinelDto := internal.BackupSentinelDto{
-		Files: NewBackupFileListBuilder().WithSimple().Build(),
+		Files: testtools.NewBackupFileListBuilder().WithSimple().Build(),
 	}
 
 	files := internal.GetRestoredBackupFilesToUnwrap(sentinelDto)
-	assert.Contains(t, files, SimplePath)
+	assert.Contains(t, files, testtools.SimplePath)
```

Comment:
SimplePath used only in tests of backup_fetch_handler. What is the reason to move it to testtools package? Also "testtools" is strange package because package should be independent and reusable entity. https://medium.com/rungo/everything-you-need-to-know-about-packages-in-go-b8bac62b74cc